README.md

```diff
@@ -412,7 +412,18 @@ These are the available config options for making requests. Only the `url` is re
 
   // `maxRedirects` defines the maximum number of redirects to follow in node.js.
   // If set to 0, no redirects will be followed.
-  maxRedirects: 5, // default
+  maxRedirects: 21, // default
+
+  // `beforeRedirect` defines function that will be called before redirect.
+  // Use this to adjust the request options upon redirecting,
+  // to inspect the latest response headers,
+  // or to cancel the request by throwing an error
+  // If maxRedirects is set to 0, `beforeRedirect` is not used.
+  beforeRedirect: (options, { headers }) => {
+    if (options.hostname === "example.com") {
```

Comment:
Just wanted to understand, why have we hardcoded the hostname here?

Reply:
this is just an example in `readme.md` on how the function can be used, in this example, we are checking if our request is redirected to `example.com` and then set the headers before proceeding.